src/Pickles/Pickles.TestFrameworks.UnitTests/NUnit/NUnit2/WhenParsingNUnitResultsFile.cs

```diff
@@ -71,6 +71,24 @@ public new void ThenCanReadInconclusiveScenarioResultSuccessfully()
         }
 
         [Test]
+        public new void ThenCanReadFeatureWithMatchingNameAndNoCategoriesCorrectly()
+        {
+            base.ThenCanReadFeatureWithMatchingNameAndNoCategoriesCorrectly();
+        }
+    
+        [Test]
+        public new void ThenCanReadFeatureWithMatchingNameAndCategoryCorrectly()
+        {
+            base.ThenCanReadFeatureWithMatchingNameAndCategoryCorrectly();
+        }
+    
+        [Test]
+        public new void ThenCanReadFeatureWithMatchingNameAndUnknownCategoryCorrectly()
+        {
+            base.ThenCanReadFeatureWithMatchingNameAndUnknownCategoryCorrectly();
```

Comment:
Note that in this case, we expect to match the first xml element encountered that has the same description as the feature. This matches the previous behavior, so that we don't break the tool for users who rely on nunit results that don't have a <Category> for each tag present in the gherkin.